it/controllers/APIControllerISpec.scala

```diff
@@ -360,7 +360,7 @@ class APIControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock wit
       s"return OK (${Status.OK})" when {
         "the get penalty details call succeeds" in {
           enableFeatureSwitch(UseAPI1812Model)
-          mockStubResponseForGetPenaltyDetailsv3(Status.OK, "123456789", body = Some(getPenaltyDetailsJson.toString()))
+          mockResponseForGetPenaltyDetailsv3(Status.OK, "123456789", body = Some(getPenaltyDetailsJson.toString()))
```

Comment:
This can be changed to 
```suggestion
          mockResponseForGetPenaltyDetails(Status.OK, "123456789", body = Some(getPenaltyDetailsJson.toString()))
```